microceph/ceph/config.go

```diff
@@ -367,3 +370,17 @@ func getMonitorAddresses(configs map[string]string) []string {
 	}
 	return monHosts
 }
+
+// formatIPv6 returns a slice in which all IPv6 addresses are formatted with square brackets.
+func formatIPv6(addrs []string) []string {
+	formatted := []string{}
+	for _, addr := range addrs {
+		if strings.Contains(addr, ":") && !strings.HasPrefix(addr, "[") && !strings.HasSuffix(addr, "]") {
```

Comment:
something like this would be more conclusive:
```
ip := net.ParseIP(addr)
if ip != nil && strings.Contains(addr, ":") {
    addr = fmt.Sprintf("[%s]", addr)
}
```